runtime/pom.xml

```diff
@@ -123,7 +130,7 @@
                             </tags>
                         </to>
                         <container>
-                            <mainClass>org.apache.karaf.minho.boot.Main</mainClass>
+                            <mainClass>org.eclipse.osc.runtime.OscMain</mainClass>
```

Comment:
It means the OscMain will be used only in Docker container. Maybe worth to add in the runtime MANIFEST (Main-Class) in maven-jar-plugin to be able to do `java -jar runtime-1.0-SNAPSHOT.jar`.